README.md

```diff
@@ -53,7 +53,7 @@ make CRYPTO=MbedTLS apps
 ## Linux
 #### Prerequisites
 ```sh
-brew cask install docker
```

Comment:
@rwalker-com these prerequisites are intended for compiling for Linux using Docker on Darwin as a host. So that is why you see the only dependency as installing docker on OSX and the rest of the toolchain and the dependencies are already set in the https://github.com/apple/HomeKitADK/blob/master/Build/Docker/Dockerfile

Reply:
interesting. ok. is support for building natively on Linux in plan?

Reply:
interesting. ok. is support for building natively on Linux in plan?

Reply:
Most of the pieces to build natively on linux are there. You just need to install the dependencies correctly, like you did in this PR.